src/obfuscation.ts

```diff
@@ -4,7 +4,7 @@ import * as SparkMD5 from 'spark-md5';
 import { PrecomputedFlag } from './interfaces';
 
 export function getMD5Hash(input: string, salt = ''): string {
-  return new SparkMD5().append(salt).append(input).end();
+  return new SparkMD5().appendBinary(salt).append(input).end();
```

Comment:
Implementing the suggested change in this comment https://github.com/Eppo-exp/js-sdk-common/pull/178#discussion_r1883114366

I wasn't finding a match with the obfuscated flags in the edge compute response until I changed I added this suggested change for how we do the md5 hash

Reply:
Okay, I can change this back to .append